### ISSUE TITLE
Colour Contrast - View Website Information page

### ISSUE THREAD

**Issue by irwink** [wet-boew/web-reporting#32]
View Website Information page
http://publiservice.tbs-sct.gc.ca/ws-nw/wr-rw/editor/website_view.aspx?lang=eng
Text inside greyed out boxes (e.g. website name) is hard to read and fails colour contrast.
Text colour #ACA899, background colour #EBEBE4, contrast ratio 1.99:1
